cli/api.py

```diff
@@ -170,10 +226,12 @@ def poll_upload_merge_status(job_id: int) -> MergeStatus:
     """Check the merge status of an upload job"""
     url = _url(f"/ingestion/poll_upload_merge_status")
     params = dict(id=job_id)
-    response = requests.get(url, params=params, headers=_with_auth())
 
-    if response.status_code != 200:
-        raise ApiError(_error_message(response))
+    @retry_with_reauth
+    def make_request():
+        return requests.get(url, params=params, headers=_with_auth())
```

Comment:
aren't you basically wrapping all `requests.xxx`s? Is there a way maybe to DRY? I would suggest monkeypatching `requests`, though I know it's a little controversial.

Reply:
Hmm, maybe!